Dockerfile

```diff
@@ -1,53 +1,21 @@
-FROM elixir:1.10.4-alpine AS build
+FROM alpine:3.12
```

Comment:
These changes mean you cannot build the Docker image locally without doing all the steps manually. What is the reasoning with removing the multistage build?

Reply:
because we have the pipeline with all the dep caches. when you want a local version, we have to use 2 dockerfiles.

Reply:
Jepp. I saw. But can't we work with Docker caching instead? This complicates IMO the workflow a bit. Not sure...

Reply:
i think a cache, based on single packages (node or elixir) is more powerful than a cache based on docker layers.